sdk/core/src/CommonNetworkMember/BaseNetworkMember/AnchoringService.ts

```diff
@@ -0,0 +1,80 @@
+import { ParsedOptions } from '../../shared/getOptionsFromEnvironment'
+import { extractSDKVersion } from '../../_helpers'
+import { ELEM_DID_METHOD, JOLO_DID_METHOD } from '../../_defaultConfig'
+import { RegistryApiService } from '@affinidi/internal-api-clients'
+import { KeysService } from '@affinidi/common'
+
+export class AnchoringService {
+  constructor(
+    private readonly encryptedSeed: string,
+    private readonly password: string,
+    private readonly didDocument: any,
+    private readonly nonce: number,
+    private readonly options: ParsedOptions,
```

Comment:
We don't need the entire `ParsedOptions` object here, we only need parameters for the `RegistryApiService` constructor.

Or we could just accept an instance of registry api service here.

Reply:
![image](https://user-images.githubusercontent.com/36485067/134005354-9673052d-586a-48c5-a8af-07d7ab3e45e0.png)
copy of old code, it is huge and has a lot of things to be refactored, I feel you want to have ideal code here, not sure if we can complete this PR with such review and previous state of the code base

Reply:
but anyway, makes sense for me